aws-auth-cognito/src/main/java/com/amplifyframework/statemachine/StateMachine.kt

```diff
@@ -103,6 +108,7 @@ internal open class StateMachine<StateType : State, EnvironmentType : Environmen
      * @param token identifies the listener to be removed
      */
     fun cancel(token: StateChangeListenerToken) {
+        Log.w("StateMachine", "canceling token: $token")
```

Comment:
Can we remove these logs here, or pass in a logger to StateMachine? We try not to use Android log directly.